packit_service/service/api/copr_builds.py

```diff
@@ -57,9 +64,7 @@ def get(self):
                     "build_id": build.build_id,
                     "status": build.status,
                     "chroots": [],
-                    "build_submitted_time": build.build_submitted_time.strftime(
-                        "%d/%m/%Y %H:%M:%S"
-                    ),
+                    "build_submitted_time": if_time_exists(build.build_submitted_time),
```

Comment:
Instead of `if_time_exists` you could call the function `optional_time`, so that this line reads: "build_submitted_time equals optional_time".

(This might be personal preference, but just by reading `if_time_exists` it's hard for me to guess what this function could do).